.github/workflows/merge-conflict-labeler.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   auto-labeler:
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-latest
```

Comment:
Counterpoint: now it might suddenly break if `ubuntu-latest` is updated and `eps1lon/actions-label-merge-conflict@v3` is not yet fixed to support it.

I'd rather keep it as it is now. In my experience, fixating versions almost always helps stabilize runs.